pkg/node/node.go

```diff
@@ -103,9 +106,24 @@ func (node *Node) NodeGetInfo(ctx context.Context, req *csi.NodeGetInfoRequest)
 		return nil, status.Error(codes.FailedPrecondition, err.Error())
 	}
 
+	topology := map[string]string{}
+	sasAddresses, err := getSASAddresses()
+	if err != nil {
+		klog.Infof("Error while searching for SAS HBA Addresses: %s", err)
+	}
+
+	for i, sasAddr := range sasAddresses {
+		//maximum value length 63 chars
+		topoKey := common.TopologyPrefix + "/" + "sas-address-" + strconv.Itoa(i)
```

Comment:
Perhaps a string length test and warning if the string is longer than 63 characters?

Reply:
I changed the way this string is built, but did not add a length check at this time. My thinking is we can skip such a check - given the current prefix/name values the only way to get beyond 63 character length would be for a system to have a 30 digit number worth of elements in sasAddresses.

Reply:
Okay